src/backend/common/manipulators/manipulator_base.py

```diff
@@ -187,3 +200,40 @@ def _clearCache(cls, models: List[TModel]) -> None:
         Needs to save _affected_references and the dirty flag
         TODO implement this
         """
+        all_affected_references: List[TAffectedReferences] = []
+        for model in models:
+            if model._dirty and model._affected_references:
+                all_affected_references.append(model._affected_references)
+
+        if all_affected_references:
+            defer(
+                cls._clearCacheDeferred,
+                all_affected_references,
+                _queue="cache-clearing",
+                # this does not exist in Cloud Tasks
+                # _transactional=ndb.in_transaction(),
+                _target="default",
```

Comment:
Can we not have this work happen on `default`? Part of the thing we want to clean up is having `default` handle all of the catchall stuff.

Reply:
Yeah; what's a better value to use?

Reply:
Uhhh - `tasks-io` I believe